.tekton/basic-component-pipeline.yaml

```diff
@@ -5,6 +5,26 @@ metadata:
 spec:
 
   finally:
+  - name: slack-notification
+    params:
+    - name: message
+      value: ':x: Build for <https://console.redhat.com/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/$(context.pipelineRun.name)|Konflux pipeline> (image: `$(params.output-image-repo)`, revision <https://github.com/stackrox/stackrox/$(params.revision)|$(params.revision)>) has failed.'
```

Comment:
> That won't work for failed pushes (because there is no PR number - yes, it could maybe be done with lots more text formatting, but I don't think it's worth it).

It's not necessarily that difficult. I noticed that PaC variables are just empty if you try to read them in the context when they aren't set.

Something as lame as
```
<$(params.git-url)/pull/{{pull_request_number}}|#{{pull_request_number}}>
```
could work, be handy and not too misleading in case of pushes.

Reply:
> :x: pull_request pipeline for [central-db-build-fc28x](https://console.redhat.com/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/central-db-build-fc28x) ([quay.io/rhacs-eng/central-db](http://quay.io/rhacs-eng/central-db), revision [02f9f058d5e5a2a14ea4246c19121cb1afdb9b2b](https://github.com/stackrox/stackrox/commit/02f9f058d5e5a2a14ea4246c19121cb1afdb9b2b) $(pull_request == "pull_request" && "https://github.com/stackrox/stackrox/pull/12312")) has failed.

Unfortunately, CEL expressions don't seem to work in this context.
Your suggestion might work, but would leave a link to `https://github.com/stackrox/stackrox/pull/` with a single character [#](https://github.com/stackrox/stackrox/pull/) link. IMO that adds more confusion that good.

If engineers follow the link to the commit, they will find the branch name and a link to the PR near the top: https://github.com/stackrox/stackrox/commit/02f9f058d5e5a2a14ea4246c19121cb1afdb9b2b

Reply:
I have a suggestion. If I don't forget, I'll show it after you merge this PR.

Reply:
Ok. I'll resolve this conversation since you approved.